sdk/azidentity/azure_cli_credential.go

```diff
@@ -53,8 +55,8 @@ func (o *AzureCLICredentialOptions) init() {
 
 // AzureCLICredential authenticates as the identity logged in to the Azure CLI.
 type AzureCLICredential struct {
-	s             *syncer
-	tokenProvider azureCLITokenProvider
+	mu   *sync.Mutex
```

Comment:
Why did `syncer` go away for CLI cred?

Reply:
Derp, `syncer` was deleted.

Reply:
For more context, I deleted `syncer` because it doesn't do much work and the CLI cred is the only one using it now that every other credential uses an MSAL client wrapper. Other non-MSAL creds could use it, however we don't plan to have many of those.